docs/src/main/sphinx/release/release-423.md

```diff
@@ -0,0 +1,95 @@
+# Release 423 (2 Aug 2023)
+
+## General
+
+* Add support for renaming fields with a `RENAME COLUMN` statement. ({issue}`16757`)
```

Comment:
Why do we call it field vs column? 

```suggestion
* Add support for renaming columns with a `ALTER TABLE ... RENAME COLUMN` statement. ({issue}`16757`)
```

Reply:
Because it renames fields. The syntax is `ALTER TABLE table RENAME COLUMN column.field TO new_field`

Reply:
I might be missing something here.. what is a field of a column? 

In our SQL docs for rename column we have

```
ALTER TABLE [ IF EXISTS ] name RENAME COLUMN [ IF EXISTS ] old_name TO new_name
```

Is the field stuff something else?

Reply:
oh .. I looked at #16959 now ... this new treatment for nested fields is completely undocumented at this stage so talking about it in the release notes is kinda bad. Anyway ... we need to catch up on this asap. 

@findepi @ebyhr @martint ... I think we need to do a few things:

1. Update ALTER TABLE and others around nested fields, what about CREATE TABLE ?
2. Maybe have some generic doc about nested fields to link to overall
3. Update docs for connectors that support that

Anything else you can think of? Is one of you going to work on this or can at least support me with relevant technical info and review?

Reply:
In CREATE TABLE statement, we can simply use `column row(field int, ...)`. Currently, only Iceberg connector supports the syntax. 

I can support you. Please feel free to ask me if you have questions.

Reply:
ok .. I will work on docs .. I also updated the release notes entry .. please have a look once I pushed